tests/fuzz/oss_fuzz_build.sh

```diff
@@ -92,10 +92,7 @@ function go_compile(){
     else
         go-fuzz -tags gofuzz -func="${function}" -o "${fuzzer}.a" .
         ${CXX} ${CXXFLAGS} ${LIB_FUZZING_ENGINE} -o "${OUT}/${fuzzer}" \
-            "${fuzzer}.a" \
-            "${TARGET_DIR}/lib/libgit2.a" "${TARGET_DIR}/lib/libssh2.a" \
-            "${TARGET_DIR}/lib/libz.a" "${TARGET_DIR}/lib64/libssl.a" \
-            "${TARGET_DIR}/lib64/libcrypto.a" \
+            "${fuzzer}.a" "${TARGET_DIR}/lib/libgit2.a" \
             -fsanitize="${SANITIZER}"
```

Comment:
By the byte, now we don't need to embed the entire dependency chain, we may be able to revert back to the Vanilla tooling from the CIFuzz. Let's revisit this in a future/separate PR.